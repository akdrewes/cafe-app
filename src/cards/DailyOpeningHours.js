import React from 'react'
import {Day} from './CafeCardStyles.js'

export default function DailyOpeningHours({day, time}) {
    
    return (
        <tr>
            <Day>{day}</Day>
            <td>{time}</td>
        </tr>
    )
}