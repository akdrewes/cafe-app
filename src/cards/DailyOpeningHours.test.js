import React from 'react'
import DailyOpeningHours from './DailyOpeningHours'
import renderer from 'react-test-renderer'

describe('DailyOpeningHours', () => {
    it('renders the weekdays with given opening hours', () => {
        const component = renderer.create(
            <DailyOpeningHours day="Mon" time={{open:'10:00', close: '18:00'}} />
            )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})