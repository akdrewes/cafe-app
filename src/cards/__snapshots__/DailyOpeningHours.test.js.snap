// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DailyOpeningHours renders the weekdays with given opening hours 1`] = `
<tr>
  <td
    className="CafeCardStyles__Day-sc-8yzm7y-16 icdUwG"
  >
    Mon
  </td>
  <td>
    10:00
     - 
    18:00
  </td>
</tr>
`;
